Packs/ThreatIntelligenceManagement/Scripts/IncidentsPerFeed/IncidentsPerFeed.py

```diff
@@ -0,0 +1,114 @@
+    """
+    params = generate_query_params(feed, from_date, query)
+    res = demisto.executeCommand("getIncidents", params)
+    severe_incidents_count = res[0]["Contents"]["total"]
+    return severe_incidents_count
+
+
+def generate_query_params(feed, from_date, query):
+    """Generating parameters for 'getIncidents' command according to relevant feed, date, and additional query
+        Args:
+                feed: Feed from which the incidents indicator should be from
+                from_date: The date from the incidents should be queried
+                query: Additional filters for the 'getIncidents' command
+
+                @return:
+                    A set with feed names
+        """
+    query_string = f'indicator.sourceBrands:{feed}'
+    if query:
+        query_string += f' and {query}'
+    params = {"query": query_string}
+    if from_date:
+        params.update({"fromdate": from_date})
+    return params
+
+
+def get_feeds() -> set:
+    """Return all enabled modules
+            @return:
+                A set with feed names
+    """
+    modules = demisto.getModules()  # type: ignore  # pylint: disable=E1101
+    return {module_details["brand"] for instance_name, module_details in modules.items() if  # pylint: disable=E1101
+            active_feed(module_details)}
+
+
+def active_feed(module) -> bool:
+    """Checks if module is active and if it's a feed and return a boolean accordingly
+            Args:
+                    module: Module to check if is active feed
+            @return:
+                True if the module's brand has 'feed' in it and if module 'state is 'active' else False
+    """
+    return FEED in module["brand"].lower() and module["state"] == ACTIVE
+
+
+def main():
+    feed_types = get_feeds()
+    distinct_incidents_query = demisto.args().get("query")
+    distinct_incidents = demisto.args().get("incidents_distinction_name")
+    from_date = demisto.args().get('from')
+    groups = generate_groups(feed_types, from_date, distinct_incidents_query, distinct_incidents)
+    human_readable = tableToMarkdown('Incidents count by feed', groups)
+    demisto.results({
+        'Type': entryTypes['note'],
+        'Contents': groups,
+        'ContentsFormat': formats['text'],
+        'ReadableContentsFormat': formats['markdown'],
+        'HumanReadable': human_readable
+    })
+
+
+def generate_groups(feed_types,
+                    from_date,
+                    distinct_incidents_query=None,
+                    distinction_name=None,
+                    ) -> List[Dict[str, Union[List[Dict[str, Union[str, List[int]]]], str]]]:
+    """If distinct_incidents_query is given- return the amount of incidents that match the query and the amount of
+    the remaining incidents that has indicators from those feeds.
+    If distinct_incidents_query is not given- will only return the amount
+        Args:
+               feed_types :A dict Containing feed names
```

Comment:
I actually think that you have a set comprehension in the `return` statement of your `get_feeds` function, so this would actually be a `set`.